azure-iot-hub/samples/iothub_registry_manager_module_method_sample.py

```diff
@@ -34,7 +35,11 @@
     iothub_registry_manager.delete_module(device_id, module_id)
```

Comment:
I haven't looked at these before.  Could you change the primary and secondary keys to generated uuid strings?  In the middle of the night some DRI is going to get a call because a sev 2 was created because a key is in the source.

Reply:
Please do it in the other samples also.